Makefile

```diff
@@ -0,0 +1,16 @@
+cnf ?= config.env
+include $(cnf)
+export $(shell sed 's/=.*//' $(cnf))
+# DOCKER TASKS
+# Build the container
+build: ## Build the container
+	docker build -t carsim .
+
+build-nc: ## Build the container without caching
+	docker build --no-cache -t $(APP_NAME) .
+
+run: ## Run container on port configured in `config.env`
+	docker run -i -t --rm --env-file=./config.env -p=$(PORT):$(PORT) --name="$(APP_NAME)" $(APP_NAME)
+
+test:
+	go test wheels/wheels_test.go
```

Comment:
@lisetterz This will not rerun tests which have cached results.

Reply:
also, it will only run the tests in one file, as opposed to all in the project.

Reply:
Fixed in f604a5eca9a79a1bd5792f3232bb4621f645e3c4